backends/p4tools/modules/testgen/options.h

```diff
@@ -69,6 +69,9 @@ class TestgenOptions : public AbstractP4cToolOptions {
     /// TestGen will consider these when choosing input and output ports.
     std::vector<std::pair<int, int>> permittedPortRanges;
 
+    /// Skip generating a control plane entry for the entities in this list.
+    std::set<cstring> skippedControlPlaneEntities;
```

Comment:
Can you say a bit about why you chose this type rather than just a set of normal strings e.g.?

Reply:
[controlPlaneName()](https://github.com/p4lang/p4c/blob/main/ir/declaration.h#L47) returns a `cstring` so it does seem like a more natural fit. `cstring` is fairly pervasive in the compiler, I am torn whether we should eventually get rid of it.

Reply:
SGTM